content/spin/v3/serverless-ai-api-guide.md

```diff
@@ -32,19 +32,19 @@ ai_models = ["codellama-instruct"]
 // -- snip --
 ```
 
-> Spin supports "llama2-chat" and "codellama-instruct" for inferencing and "all-minilm-l6-v2" for generating embeddings.
+> Spin supports the models of the Llama architecture for inferencing and "all-minilm-l6-v2" for generating embeddings.
 
 ### File Structure
 
-By default, the Spin framework will expect any already trained model files (which are configured as per the previous section) to be downloaded by the user and made available inside a `.spin/ai-models/` file path of a given application. For example:
+By default, the Spin framework will expect any already trained model files (which are configured as per the previous section) to be downloaded by the user and made available inside a `.spin/ai-models/` file path of a given application. The files for each of the models need to be placed under a folder named after the architecture of the model (i.e) for `llama2-chat`, the folder needs to be named `llama`. For example:
 
 ```bash
-code-generator-rs/.spin/ai-models/llama/codellama-instruct
+code-generator-rs/.spin/ai-models/llama/codellama-instruct/{safetensors | config.json | tokenizer.json}
```

Comment:
Incidentally the (unchanged) tutorial content implies `codellama-instruct` is a single file, but here you show it as a directory containing multiple files - would be good to figure out which is right (which of course may have changed!) and, depending on the outcome, either change the tutorial or use a different example model here.